global-news-app/src/Hooks/SearchContext/useUserLocation.ts

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import { useQuery } from "react-query";
+import useLocalStorage from "./useLocalStorage";
+import useSearchContext from "./useSearchContext";
+import { useEffect } from "react";
+
+export const geoLocationApiKey = "0daad5e0-82e7-11ee-92e0-f5d620c7dcb4";
+
+interface GeolocationDbResponse {
+  country_code: string;
+  country_name: string;
+}
+export const useUserLocation = () => {
+  const [location, setLocation] = useLocalStorage<GeolocationDbResponse | null>(
+    "userLocation",
+    null
+  );
+  const { updateSelection } = useSearchContext();
+
+  useEffect(() => {
+    if (location) {
+      updateSelection("country", {
```

Comment:
instead of the string pass as a parameter correct value of the `SelectionsKeys` enum